connector/src/main/scala/com/microsoft/kusto/spark/common/KustoOptions.scala

```diff
@@ -24,6 +24,9 @@ trait KustoOptions {
   val KUSTO_AAD_APP_SECRET: String = newOption("kustoAadAppSecret")
 
   val KUSTO_ACCESS_TOKEN: String = newOption("accessToken")
+
+  // Use only for local runs where one can open the browser
+  val KUSTO_USER_PROMPT: String = newOption("userPrompt")
```

Comment:
I think it'd be clearer of all of these had the suffix "authentication"

Reply:
i dont want to change these now ... last time we changed it caused a lot of problems